composer.json

```diff
@@ -19,7 +19,9 @@
         "slim/slim": "^3.11",
         "danjam/slim-mustache-view": "^1.1",
         "tuupola/slim-basic-auth": "^3.2.1",
-        "akrabat/ip-address-middleware": "^1.0.1"
+        "akrabat/ip-address-middleware": "^1.0.1",
+        "frontpack/composer-assets-plugin": "^0.10.2",
+        "twbs/bootstrap": "4.0.0"
```

Comment:
Why the fixed (and .0.0) version for Bootstrap?

Reply:
Well I just took what was used up until here in the template.
https://github.com/Rarst/release-belt/blob/5bb75c07bf773bd0f1008a119d0a0297d469542d/src/mustache/index.mustache#L7

Reply:
Can't write flexible version in plain HTML. :D Well, short of CDN magic. Cheers for update!